recipe/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "iam-units" %}
+{% set version = "2020.4.21" %}
+{% set hash = "042f298a86888aa5fe7771b85549c2f90ffc39255c00670c0ec73c058df647b5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+  - python
+  - pip
```

Comment:
It looks like you will probably need `setuptools_scm` as a `host` requirement

Reply:
Done.